dataprep/tests/eda/test_config.py

```diff
@@ -0,0 +1,35 @@
+"""
+    This module for testing config parameter
+"""
+
+from ...eda.basic.configs import BarChart, Histogram, WordCloud
+
+
+def test_hist() -> None:
+    dict_data = {"bins": 10, "agg": "mean", "value_range": [0, 1]}
+    Histogram.from_dict(dict_data)
+    for key in dict_data.keys():
+        assert Histogram.__dict__[key] == dict_data[key]
```

Comment:
Could you create a hist object rather than directly use the class? The parameter should be in an Histogram object.